setup.py

```diff
@@ -78,6 +78,7 @@ def get_version():
         "mongoengine==0.20.0",
         "motor>=2.3,<3",
         "numpy",
+        "opencv-python-headless>=4.4,<5",
```

Comment:
Is there a reason to include a version requirement at all? I'm not aware of anything we're using that has or will change anytime soon.

Reply:
I'll remove